### PR TITLE
General cleanup

### PR DESCRIPTION
**Pull request by kevmoo**
Just want to make sure I keep travis happy...

### CODE REVIEW

**Comment by kevmoo** [comment]
I'm preparing to add JSON-formatted warnings/errors – wanted to get this in first

File: lib/src/model.dart [hunk 1]
```diff
@@ -3843,21 +3581,24 @@ class Package extends Nameable implements Documentable {
...
+      messageParts
+          .add("${warnablePrefix} ${warnableStrings.name}: ${warnableStrings
+              .location}");
+    }
     if (referredFrom != null) {
       for (Locatable referral in referredFrom) {
         if (referral != warnable) {
-          Tuple2<String, String> referredFromStrings =
-              nameAndLocation(referral);
+          var referredFromStrings =
```
Comment (devoncarew) [inline lib/src/model.dart]: I'll defer to @jcollins-g here, but removing the type here makes the code strictly less readable. You get no info from the right abut what the type is, and removing it from the left leaves you w/ nothing to go on. In this case it's not just something simple like a `String`, but a more complicated generic type - `Tuple2<String, String>`. (similarly with the `final` assignment above)

Comment (kevmoo) [reply lib/src/model.dart]: in both cases, I'm explicitly initializing `new _LocatableStrings`. Not sure how this is less readable.

🤷‍♂️

Comment (devoncarew) [reply lib/src/model.dart]: Sorry, I read this as replacing `Tuple2<String, String>` with `var`, not `_LocatableStrings` with var.

**Comment by kevmoo** [comment]
@jcollins-g – when you're bored...

File: lib/src/html/html_generator_instance.dart [hunk 2]
```diff
@@ -11,6 +11,7 @@ import 'package:collection/collection.dart' show compareNatural;
 import 'package:path/path.dart' as path;
 
 import '../model.dart';
+import '../warnings.dart';
```
Comment (jcollins-g) [inline lib/src/html/html_generator_instance.dart]: yes, been meaning to do this, thanks.

File: lib/src/model.dart [hunk 3]
```diff
@@ -4674,3 +4416,20 @@ class TypeParameter extends ModelElement {
   @override
   String toString() => element.name;
 }
+
+class _LocatableStrings {
```
Comment (jcollins-g) [inline lib/src/model.dart]: I appreciate wanting to get rid of the duct tape, but I would prefer we modify Locatable's elementLocation to simply return what warnable needs (I don't think we use elementLocation for any other purposes).

File: testing/test_package/lib/fake.dart [hunk 4]
```diff
@@ -70,6 +70,7 @@ abstract class ImplementingThingy2 implements BaseThingy2, ImplementingThingy {}
 /// Test for MultiplyInheritedExecutableElement handling.
 abstract class BaseThingy2 implements BaseThingy {
   @override
+
```
Comment (jcollins-g) [inline testing/test_package/lib/fake.dart]: ?

File: pubspec.lock [hunk 5]
```diff
@@ -6,7 +6,7 @@ packages:
       name: analyzer
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.30.0"
+    version: "0.30.0+4"
```
Comment (jcollins-g) [inline pubspec.lock]: any particular reason for the upgrades?

File: lib/src/html/html_generator_instance.dart (hunk 2 above)
Comment (kevmoo) [reply lib/src/html/html_generator_instance.dart]: 👍

File: pubspec.lock (hunk 5 above)
Comment (kevmoo) [reply pubspec.lock]: get the latest markdown, gets rid of which/when – which aren't used.

File: testing/test_package/lib/fake.dart (hunk 4 above)
Comment (kevmoo) [reply testing/test_package/lib/fake.dart]: dartfmt oops, I guess...fixed

File: lib/src/model.dart (hunk 3 above)
Comment (kevmoo) [reply lib/src/model.dart]: Fixed.

**Comment by kevmoo** [comment]
@jcollins-g PTAL

**Pull request merged**